src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/config/OrchestrationExceptionHandler.kt

```diff
@@ -126,14 +127,20 @@ data class ErrorResponse(
   val errorCode: Int? = null,
   val userMessage: String? = null,
   val developerMessage: String? = null,
-  val moreInfo: String? = null,
 ) {
   constructor(
     status: HttpStatus,
     errorCode: Int? = null,
     userMessage: String? = null,
     developerMessage: String? = null,
-    moreInfo: String? = null,
   ) :
-    this(status.value(), errorCode, userMessage, developerMessage, moreInfo)
+    this(status.value(), errorCode, userMessage, developerMessage)
 }
+
+data class ApplicationValidationErrorResponse(
```

Comment:
Do we want to use inheritance here as well? To keep consistent with the other changes made

data class ApplicationValidationErrorResponse(
  val validationErrors: List<ApplicationValidationErrorCodes>,
) : ErrorResponse(status = HttpStatus.UNPROCESSABLE_ENTITY)